Marlin/configurator/config/language.h

```diff
@@ -167,7 +167,8 @@
 #define MSG_ENDSTOP_OPEN                    "open"
 #define MSG_HOTEND_OFFSET                   "Hotend offsets:"
 
-#define MSG_SD_CANT_OPEN_SUBDIR             "Cannot open subdir"
+#define MSG_SD_CANT_OPEN_SUBDIR             "Cannot open subdir: "
```

Comment:
@thinkyhead no problem for me (not using any of the SD card functions from the host). But might cause problems for octoprint, as I think @foosel is using the SD card functions.